docs/content/latest/quick-start/build-apps/csharp/ysql-entity-framework.md

```diff
@@ -0,0 +1,313 @@
+(2 rows)
+```
+
+```sh
+ysql_entityframework=# SELECT count(*) FROM products;
+```
+
+```output
+ product_id |    description    | price | product_name
+------------+-------------------+-------+--------------
+          1 | 200 page notebook |  7.50 | Notebook
+          2 | Mechanical pencil |  2.50 | Pencil
+(2 rows)
+```
+
+Create 2 orders with products using the `userId` for John.
+
+```sh
+$ curl \
+  --data '{ "userId": "1", "products": [ { "productId": 1, "units": 2 } ] }' \
+  -v -X POST -H 'Content-Type:application/json' http://localhost:8080/orders
+```
+
+```sh
+$ curl \
+  --data '{ "userId": "1", "products": [ { "productId": 1, "units": 2 }, { "productId": 2, "units": 4 } ] }' \
+  -v -X POST -H 'Content-Type:application/json' http://localhost:8080/orders
+```
+
+## Query results
+
+### Using the YSQL shell
+
+```sh
+$ ./bin/ysqlsh
+```
+
+```output
+ysqlsh (11.2)
+Type "help" for help.
+yugabyte=#
+```
+
+```sh
+yugabyte=# \c ysql_entityframework
+```
+
+```output
+You are now connected to database "ysql_entityframework" as user "yugabyte".
+```
+
+```sql
+ysql_entityframework=# SELECT count(*) FROM users;
+```
+
+```output
+ count
+-------
+     2
+(1 row)
+```
+
+```sql
+ysql_entityframework=# SELECT count(*) FROM products;
+```
+
+```output
+ count
+-------
+     2
+(1 row)
+```
+
+```sql
+ysql_entityframework=# SELECT count(*) FROM orders;
+```
+
+```output
+ count
+-------
+     2
+(1 row)
+```
+
+### Using the REST API
+
+```sh
+$ curl http://localhost:8080/users
+```
+
+```json
+{
+  "content": [
+    {
+      "userId": 2,
+      "firstName": "Tom",
+      "lastName": "Stewart",
+      "email": "tstewart@example.com"
+    },
+    {
+      "userId": 1,
+      "firstName": "John",
+      "lastName": "Smith",
+      "email": "jsmith@example.com"
+    }
+  ],
+  ...
+}
+```
+
+```sh
+$ curl http://localhost:8080/products
+```
+
+```json
+{
+  "content": [
+    {
+      "productId": 2,
+      "productName": "Pencil",
+      "description": "Mechanical pencil",
+      "price": 2.5
+    },
+    {
+      "productId": 1,
+      "productName": "Notebook",
+      "description": "200 page notebook",
+      "price": 7.5
+    }
+  ],
+  ...
+}
+```
+
+```sh
+$ curl http://localhost:8080/orders
+```
+
+```json
+{
+  "content": [
+    {
+      "orderTime": "2019-05-10T04:26:54.590+0000",
+      "orderId": "999ae272-f2f4-46a1-bede-5ab765bb27fe",
+      "user": {
+        "userId": 1,
+        "firstName": "John",
+        "lastName": "Smith",
+        "email": "jsmith@example.com"
+      },
+      "userId": null,
+      "orderTotal": 25,
+      "products": []
+    },
+    {
+      "orderTime": "2019-05-10T04:26:48.074+0000",
+      "orderId": "1598c8d4-1857-4725-a9ab-14deb089ab4e",
+      "user": {
+        "userId": 1,
+        "firstName": "John",
+        "lastName": "Smith",
+        "email": "jsmith@example.com"
+      },
+      "userId": null,
+      "orderTotal": 15,
+      "products": []
+    }
+  ],
+  ...
+}
+```
```

Comment:
This is the old output. I have updated the new one in the doc(there are some differences with the format and the fields that are output). Other than this, LGTM.

Reply:
Thanks @Sfurti-yb , I'll update it.